config/fcos/v1_6_exp/translate.go

```diff
@@ -133,6 +133,10 @@ func (c Config) processBootDevice(config *types.Config, ts *translate.Translatio
 		wantEFIPart = true
 	case *layout == "ppc64le":
 		wantPRePPart = true
+	case *layout == "s390x-virt":
+		wantBIOSPart = true
+		wantEFIPart = true
```

Comment:
Okay, I think I added this because Benjamin suggested that KVM Virt does not need `/dev/vda` as `/dev/disk/by-partlabel` similar to `x86`. I'll correct by removing this case statement and append following case statement.

`case *layout == "s390x-eckd" || *layout == "s390x-zfcp" || *layout == "s390x-virt"`

Reply:
Initially I used `device` but Steve suggested something specific to `s390x` that's why I added `s390x-device`. I'll correct that to `device`.

Reply:
> I'll correct by removing this case statement and append following case statement.

Yes, that makes sense to me.

> Initially I used `device` but Steve suggested something specific to `s390x` that's why I added `s390x-device`. I'll correct that to `device`.

Yeah, I think it's debatable either way but I would lean towards `device`. Having to repeat "s390x" in both the layout name and this field's name is a bit awkward.